tests/locustfile.py

```diff
@@ -0,0 +1,23 @@
+"""
+Stress Test AWS Lambda server through API Gateway
+"""
+import json
+
+from locust import HttpUser, between, task
```

Comment:
did you make sure to add locust to the requirements.txt file?

Reply:
I added locust to the requirements.txt